src/components/BookCreate.js

```diff
@@ -1,83 +1,78 @@
 import React, {useState} from 'react';
-import {postBook} from '../utils/postBook';
-import BookAddSuccess from './BookAddSuccess';
+import documentClient from '../configuredDocumentClient';
 
-const BookCreate = (props) => {
-    const [bookState, setBookState] = useState({
-        bookTitle: '',
-        complete: false,
-        isbnNumber: '',
+const BookCreate = ({match, history}) => {
+    const [book, setbook] = useState({
+        bookId: match.params.id,
+        title: '',
+        isbn: '',
         shelf: '',
-        loggedInName: props.loggedInName,
     });
 
-    const setState = (state) => setBookState({...bookState, state});
-
     const handleInputChange = (event) => {
         const target = event.target;
         const value = target.value;
         const name = target.name;
 
-        setState({
-            [name]: value
+        setbook((prevState) => {
+            return {
+                ...prevState,
+                [name]: value
+            };
         });
     }
 
-    const handleSubmit = (event) => {
+    const handleSubmit = async (event) => {
```

Comment:
Maybe something to think about in another PR, but do we want to have some safety here to keep a submission from occurring when all fields are not filled out before submission?

Reply:
Yea, we should mark the fields required so the form won't submit without them present